src/applications/vaos/appointment-list/components/AppointmentsPageV2/AppointmentListItem.jsx

```diff
@@ -69,85 +69,98 @@ function VAFacilityName({ facility }) {
 }
 
 export default function AppointmentListItem({ appointment, facility }) {
+  const history = useHistory();
   const appointmentDate = moment.parseZone(appointment.start);
   const facilityId = getVARFacilityId(appointment);
   const isCommunityCare = appointment.vaos.isCommunityCare;
   const isVideo = appointment.vaos.isVideo;
   const isPhone = isVAPhoneAppointment(appointment);
   const isInPersonVAAppointment = !isVideo && !isCommunityCare && !isPhone;
   const canceled = appointment.status === APPOINTMENT_STATUS.cancelled;
+  const link = isCommunityCare
+    ? `cc/${appointment.id}`
+    : `va/${appointment.id}`;
 
   return (
     <li
       data-request-id={appointment.id}
-      className="vaos-appts__card vads-u-display--flex vads-u-align-items--center"
+      className="vaos-appts__card vaos-appts__card--clickable"
```

Comment:
You might consider removing the padding from this element and adding it to the div below. The way it is now, the whole card isn't clickable, just the inner area immediately surrounding the text.

Green area isn't clickable:
![Screen Shot 2021-04-08 at 2 03 16 PM](https://user-images.githubusercontent.com/634932/114075139-31da8e80-9873-11eb-9714-3a59d78e0beb.png)